docs/camino-messenger/introduction.md

```diff
@@ -216,7 +217,26 @@ In the Camino Messenger Protocol, there is no static data inclusion in any of th
+The V1Alpha version first release was done on 18 January 2023. Feedback and reviews with Camino Network Partners are leading to improvements of Message Types. A partner working with a specific Message Type to implement a connection to their system discovers errors or finds a concept is missing in the Camino MessageType. Conclusively one Message Type can be mature and and have only non-breaking improvements, while another Message Type is actively being developed. While "Accommodation" might be in a productive V1, we might have "Car Rental" in a V2Beta. As all message formats are released together, we simply have numeric releases that contain updates on one or more messages from one or more feature branches and PRs. Unless a partner is working together with us to develop a new message or new features in an existing message, the c4t (main) branch should be used.
+
+### Version of each Message Type
+
+The version of each Message Type is specified in the package name stored on [buf.build](https://buf.build/chain4travel/camino-messenger-protocol/docs) . The package name and conclusively the version is included in the gRPC header. To define the supported capabilities for the partner configuration, we can use this package name as a whole to refer to a service as a capability. For example: cmp.services.accommodation.v1alpha.AccommodationSearchService
+
+We have decided to step away from semantic versioning and consider all changes in the protocol a breaking change. Conclusively, partners can only trade using the exact same message version.
+
+### Camino Messenger Bot releases and number of versions supported
+
+Every time a new version is released of any of the message types a new version of the bot is released. The latest version of the Camino Messenger bot can be downloaded from our [github](https://github.com/chain4travel/camino-messenger-bot). The main bot version is the same as the latest protocol version. Technical releases for the bot can result to a minor version increase (for example bot release 6.0.0 and 6.1.0 which both support latest protocol version 6, but bot release 6.1.0 has a technical enhancement without an update to the protocol).
+
+The bot will always support the latest version of a message type and the previous version of that same message type. The version before the previous version will still be in the SDK on buf.build, so that in the transition phase after the release of a new version for a specific Message Type, for example Partners A and B can still be trading on protocol version 3 (for example using Bot release 4.0.0), while partners C and D are trading on protocol version 4 (using the same bot release 4.0.0) and partners E and F are already trading on protocol version 5 of that message type using the most recent bot release 5.0.0. Upon release of the new Bot release 6.0.0, partners trading on version 3 of the message type will receive an "obsolete" warning with each message.
```

Comment:
I feel like this paragraph is not clear enough to understand. The issue I have with it is that we're talking about 3 different versions here:
* Bot version (Synced with the protocol-**release**)
* Protocol release
* Service versions 

Maybe we should find a better name for the service versions? They have been mentioned in the first paragraph as "The version of each Message Type", so maybe "message type version"?

With that being said this paragraph should be very clear about which version is meant, because I feel like it's very hard to tell if for instance "protocol version 3" is referring to a message type version or the protocol release.

Reply:
I agree that it's better to use "_service version_" instead of "_message type version_". Because every message definition in Protobuf is a _message type_. It's a widely used term, so, it's easy to get confused by it.